tools/sicer/sicer_wrapper.xml

```diff
@@ -28,6 +28,7 @@
   </command>
   <requirements>
     <requirement type="package" version="1.1">SICER</requirement>
+    <requirement type="package" version="1.1.0">scipy</requirement>
```

Comment:
That's actually not needed since the sicer bioconda package already depends on scipy: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/sicer/meta.yaml#L27

Reply:
I think it's because it never uses the SICER bioconda package. It installs the toolshed package and the bioconda package but it doesn't activate the SICER conda environment. So the scipy is missing in a clean Galaxy instance.

Reply:
That's because the `tool_dependencies.xml` file was present in the ToolShed repository, once removed this will work fine.